### DISCUSSION TITLE
Error: oauth2: cannot fetch token: 404 Not Found, only with fly login and works fine with all browsers

### DISCUSSION THREAD

**Discussion by arungskumar** [concourse/concourse#5956] [General]
## Summary
After a successful install of concourse using bosh, able to login using basic-auth username and password on the browser. But, getting below 404 error when tried logging in using fly.

## Steps to reproduce

- Bosh install, with below manifest instance group

_name: web
    instances: 1
    azs: ((azs))
    networks: [{name: ((network_subnet_name))}]
    static_ips: ((external_host))
    stemcell: xenial
    vm_type: Standard_DS1_v2
    jobs:
      - name: web
        release: concourse
        properties:
          redact_secrets: true
          tls:
            bind_port: 443
            cert: ((atc_tls))
          token_signing_key: ((token_signing_key))
          external_url: https://((external_host))
          postgresql:
            database: &db_name atc
            role: &db_role
              name: concourse
              password: ((postgres_password))
            require_tls: false
          worker_gateway:
            host_key: ((tsa_host_key))
            authorized_keys: |
              ((worker_key.public_key))
          add_local_users: ['admin:test123']
          main_team:
            auth:
              local:
                users: [admin]_

## Expected results
Successful login using `fly`

## Actual results
_logging in to team 'main'

2020/08/05 11:16:38 GET /api/v1/info HTTP/1.1
Host: concourse.domain-name.com
User-Agent: Go-http-client/1.1
Accept-Encoding: gzip

2020/08/05 11:16:38 HTTP/1.1 200 OK
Content-Length: 102
Content-Type: application/json
Date: Wed, 05 Aug 2020 18:16:38 GMT
Vary: Accept-Encoding
X-Concourse-Version: 6.4.1
X-Content-Type-Options: nosniff
X-Download-Options: noopen
X-Frame-Options: deny
X-Xss-Protection: 1; mode=block

{"version":"6.4.1","worker_version":"2.2","external_url":"https://concourse.domain-name.com"}

WARNING:

fly version (5.4.1) is out of sync with the target (6.4.1). to sync up, run the following:

    fly -t test sync

error: oauth2: cannot fetch token: 404 Not Found
Response: 404 page not found_

## Additional context
Enabled debug logging and looked into logs, was not able to see any error regarding this.

## Triaging info

* Concourse version: 6.4.1
* Browser (if applicable): chrome/safari/firefox
* Did this used to work? New install and works only on the browser.

**Comment by jamieklassen** [comment]
there are backwards-incompatible auth changes between 5.4.1 (which seems to be the version of fly you have) and 6.4.1 -- can you try re-downloading fly (from the 6.4.1 github release, or the web UI, or the `/api/v1/cli` endpoint) and then logging in?

**Reply by arungskumar** [reply]
Sure @jamieklassen , will this downgrade option and let you know.

**Reply by arungskumar** [reply]
Thanks that helped, appreciate your help.
